eve/main.yml

```diff
@@ -498,6 +498,19 @@ stages:
           command: docker push ${DEVELOPMENT_DOCKER_IMAGE_NAME}
           haltOnFailure: true
           env: *docker_env
+      - ShellCommand:
+          name: Oras login
+          command: |
+            oras login --username "${HARBOR_LOGIN}" --password "${HARBOR_PASSWORD}" registry.scality.com
+          env:
+            HARBOR_LOGIN: '%(secret:harbor_login)s'
+            HARBOR_PASSWORD: '%(secret:harbor_password)s'
+      - ShellCommand:
+          name: push dashboards to the development namespace
+          command: |
+            ./push-to-registry.sh cloudserver-dashboard:latest dashboard.json "application/grafana-dashboard+json"
```

Comment:
looking at the script: not sure it was really worth the effort, calling `oras` directly is just as short/long as calling the script...

Reply:
it is easier to copy/paste it in other repos (vault, backbeat).
if we need to push more files it makes it easier I think.

Reply:
once we are moved to github-actions, we may be able to write a dedicated and re-usable "oras" github action, which will take care of that